lib/llvmopencl/Workgroup.cc

```diff
@@ -1230,7 +1235,12 @@ Workgroup::createAllocaMemcpyForStruct(LLVMModuleRef M, LLVMBuilderRef Builder,
     args[1] = CARG1;
     args[2] = Size;
 
+#ifdef LLVM_OLDER_THAN_14_0
     LLVMValueRef call1 = LLVMBuildCall(Builder, MemCpy1, args, 3, "");
```

Comment:
same

Reply:
yes, i will fix ..
There are a couple other similar deprecated api's that need similar handling for 14